sauce/features/budget/display-goal-amount/index.js

```diff
@@ -25,18 +25,19 @@ export default class DisplayTargetGoalAmount extends Feature {
       const targetBalance = subCategory.get('targetBalance');
       const targetBalanceDate = monthlySubCategoryBudgetCalculation.get('goalTarget');
       if (goalType === 'MF') {
-        $('#' + emberId + '.budget-table-row.is-sub-category div.budget-table-cell-goal').text('$' + monthlyFunding / 1000);
+        $('#' + emberId + '.budget-table-row.is-sub-category div.budget-table-cell-goal').text('$' + ynab.YNABSharedLib.currencyFormatter.format(monthlyFunding));
```

Comment:
The dollar sign should not be hardcoded so the feature will work for all supported currencies. The currency symbol can be obtained like this; `ynab.YNABSharedLib.currencyFormatter.getCurrency().currency_symbol`.

I see you aren't using `.convertToMilliDollars(value)` but in your previous code you were dividing the amount by 1000. The convert routine does that for you assuming you still need it.